src/6.6/6.6.3.0.md

```diff
@@ -0,0 +1,107 @@
+---
+nav:
+  title: v6.6.3.0
+meta:
+  date: "2024-05-08"
+---
+# Release notes Shopware 6.6.3.0
+
+## Abstract
+
+Besides a list of 61 bug fixes, this minor release contains some cool improvements. The new feature Shopping list was released.
+
+## System requirements
+
+* tested on PHP 8.2 and 8.3
+* tested on MySQL 8 and MariaDB 10.11
+
+## Improvements
+
+### Typescript support for webpack config in extensions
+
+Extensions can now use typescript (.ts and .cts) files for their webpack configuration.
+
+### Added meteor component library to the administration
+
+The [Meteor component library](https://shopware.design/meteor-components/) was added as a new dependecy to the administration. All library components are now also directly available in the administration.
+
+### Shopping List feature release.
+
+As Business Partner create and manager your Shopping Lists.
+
+## Fixed bugs
+
+*  [NEXT-34787 - show-correct-saleschannel-on-customer-view.md](./changelog/release-6-6-2-0/2023-03-28-show-correct-saleschannel-on-customer-view.md) ([Fabian Blechschmidt](https://github.com/Schrank))
+*  [NEXT-33970 - Changed error handling for ImportExport](./changelog/release-6-6-2-0/2024-03-12-changed-error-handling-for-import-export.md)
```

Comment:
Could you please verify the fixed bugs? I saw this one was released in 6.6.2.0 already.

Reply:
We also need to link the ticket to our issue tracker https://issues.shopware.com/

Reply:
Hi, I just compared the changes from saas/2024/18 and saas/2024/19 in the platform and summarized them in the list.
With the Plugin, I guess the Shopping List feature needs to be included in the release of the notes.
By issue tracker, how do we get the change from there?

Reply:
@thuanlv287, just align the format with the latest mentioned tickets below (like `NEXT-29683`)